docker/centos/Dockerfile

```diff
@@ -1,19 +1,35 @@
 FROM centos:7
 
-RUN yum install -y sudo \
-    && yum clean all \
-    && localedef -i en_US -f UTF-8 en_US.UTF-8 \
-    && useradd -m -s /bin/bash am-prompt \
-    && echo 'am-prompt ALL=(ALL) NOPASSWD:ALL' >>/etc/sudoers
+RUN set -ex; \
+    yum install -y sudo; \
+    yum clean all; \
+    localedef -i en_US -f UTF-8 en_US.UTF-8; \
+    useradd -m -s /bin/bash am-prompt; \
+    echo 'am-prompt ALL=(ALL) NOPASSWD:ALL' >>/etc/sudoers
 
 USER am-prompt
 WORKDIR /home/am-prompt
 
 ENV LANG=en_US.UTF-8 \
-    USER=am-prompt
+    USER=am-prompt \
+    AM_PROMPT_UPDATE=0
 
 COPY . ./
-RUN ./install.sh
 
-SHELL [ "/bin/bash", "-l" ]
-RUN git-sha
+RUN set -ex; \
+    ./install.sh 'bash';
+
+SHELL [ "/home/linuxbrew/.linuxbrew/bin/bash", "-lc" ]
+
+RUN set -ex; \
+    echo "$SHELL" | grep 'bash'; \
+    git-sha
+
+RUN set -ex; \
+    ./install.sh 'zsh';
+
+SHELL [ "/home/linuxbrew/.linuxbrew/bin/zsh", "-lc" ]
```

Comment:
@dmccaffery looks like `zsh` was never installed; I'm pretty sure this is why $SHELL is blank
This is true for all the linux variants